docs/release.md

```diff
@@ -118,3 +118,35 @@ repo][amazon].
 
 [amazon]: https://github.com/aws-quickstart/quickstart-heptio
 
+### 7. Certify the Cluster
+
+Once Amazon has merged the PR, you can certify the cluster. Certification gets
```

Comment:
Before it's merged, we should certify.

Reply:
Or at least ensure all the tests are passing.

Reply:
Step 5 already involves a Sonobuoy pass. Is that not sufficient?